_dashboards/quickstart.md

```diff
@@ -1,120 +1,119 @@
 ---
 layout: default
-title: Quickstart guide
+title: OpenSearch Dashboards quickstart
 nav_order: 2
 has_children: false
 redirect_from:
    - /dashboards/quickstart-dashboards/
 ---
 
-# Quickstart guide
+# OpenSearch Dashboards quickstart
 
-This quickstart guide covers the core concepts that you need to understand to get started with OpenSearch Dashboards. You'll learn how to:
+This quickstart covers the following basics to help you get started with OpenSearch Dashboards:
 
-- Add sample data.
-- Explore and inspect data.
-- Visualize data.
+- **Adding sample data:** Use pre-loaded visualizations, dashboards, and other tools to help you explore OpenSearch Dashboards before you add your own data.
+- **Exploring your data:** Dig into your data and see what is there.
+- **Creating visualizations:** Make your data more accessible by creating visualizations.
 
-Here's a glance at the view you see when you open the **Dashboard** or **Discover** tool.
+Your OpenSearch Dashboards view can be in either [light or dark mode](#turning-on-and-off-dark-mode), depending on your admin settings. To dock or undock the navigation pane, select the menu icon {::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/menu-icon.png" class="inline-icon" alt="menu icon"/>{:/} and then **Dock navigation** or **Undock navigation**. A view of the OpenSearch Dashboards homepage is shown in the following image.  
 
-<img src="{{site.url}}{{site.baseurl}}/images/dashboards/new-look.png" alt="Light and dark mode UI on Discover and Dashboard tools' home page" width="700">
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/opensearch-dashboards-home.png" alt="OpenSearch Dashboards homepage" width="700">
 
 {::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/alert-icon.png" class="inline-icon" alt="alert icon"/>{:/} **Note**<br>Before you get started, make sure you've installed OpenSearch and OpenSearch Dashboards. For information about installation and configuration, see [Install and configure OpenSearch]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/index/) and [Install and configure OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/install-and-configure/install-dashboards/index/).
 {: .note}
 
-## Adding sample data
+The following sections include tutorials on using OpenSearch Dashboards applications and tools. You can use your own environment or the [OpenSearch Playground](https://playground.opensearch.org/app/home#/), which runs on the latest version of OpenSearch. 
```

Comment:
By OpenSearch Playground "runs on the latest version of OpenSearch", do we mean the latest numbered version or does Playground run a future version?

Reply:
The latest numbered version. 
![Uploading Screenshot 2024-02-26 at 10.23.08 AM.png…]()